aiohttp/web_response.py

```diff
@@ -268,6 +271,41 @@ def last_modified(
         elif isinstance(value, str):
             self._headers[hdrs.LAST_MODIFIED] = value
 
+    @property
+    def etag(self) -> Optional[ETag]:
+        quoted_value = self._headers.get(hdrs.ETAG)
+        if not quoted_value:
+            return None
+        elif quoted_value == ETAG_ANY:
+            return ETag(value=ETAG_ANY)
+        match = QUOTED_ETAG_RE.match(quoted_value)
```

Comment:
Maybe `fullmatch()`?
```suggestion
        match = QUOTED_ETAG_RE.fullmatch(quoted_value)
```

Reply:
Fixed, thanks.